components/landing/TheHeroMoment.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <section class="the-hero-moment">
+    <article class="the-hero-moment__content">
+      <VersionInfo class="the-hero-moment__version-info" />
+      <h1 class="the-hero-moment__title">
+        Open-Source Quantum Development
+      </h1>
+      <p class="the-hero-moment__description">
+        Qiskit [kiss-kit] is an open source SDK for working with quantum
+        computers at the level of pulses, circuits and algorithms.
+      </p>
+      <LandingCta
+        class="the-hero-moment__cta"
+        href="https://qiskit.org/documentation/install.html"
+      >
+        Get started
+      </LandingCta>
+    </article>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import VersionInfo from './VersionInfo.vue'
+import LandingCta from './LandingCta.vue'
+
+@Component({
+  components: { VersionInfo, LandingCta }
+})
+export default class extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+@import '~carbon-components/scss/globals/scss/typography';
+
+.the-hero-moment {
+  position: relative;
+  display: flex;
+  padding: 0 $spacing-05;
+
+  &__picture {
```

Comment:
Are you using this on the code? I don't see where. If you aren't using it, please remove it :)